.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "openpilot devcontainer",
+  "initializeCommand": "docker build -t openpilot_base -f Dockerfile.openpilot_base .",
+  "build": {
+    "context": "..",
+    "dockerfile": "../Dockerfile.openpilot_dev"
+  },
+  "privileged": true,
+  "containerEnv": {
+    "DISPLAY": "${localEnv:DISPLAY}",
+    "force_color_prompt": "1",
+    "PYTHONPATH": "/workspaces/openpilot"
+  },
+  "runArgs": [
+    "--volume=/tmp/.X11-unix:/tmp/.X11-unix:rw",
+    "--volume=${localEnv:XAUTHORITY}:/root/.Xauthority"
```

Comment:
Additional local fs mountpoints, which are useful to persist:
* `$HOME/.comma` => `/root/.comma ` - stores auth tokens 
* `/tmp/comma_download_cache` => `/tmp/comma_download_cache` - `COMMA_CACHE`